scalars/contributed/Juke-Duke/email-address.md

```diff
@@ -0,0 +1,89 @@
+- Uppercase and lowercase Latin letters `A` to `Z` and `a` to `z`
+- Digits `0` to `9`
+- Printable characters ``!#$%&'*+-/=?^_`{|}~``
+- Dot `.`, provided that it is not the first or last character and provided also that it does not appear consecutively
+
+The domain part may contain any of the following ASCII characters:
+- Uppercase and lowercase Latin letters `A` to `Z` and `a` to `z`
+- Digits `0` to `9`, provided that top-level domain names are not all-numeric (.com, .net, .org, etc.)
+- Hyphen `-`, provided that it is not the first or last character
+
+**Examples**
+
+The general format is described in [RFC5322 Section 3.4.1](https://www.rfc-editor.org/rfc/rfc5322#section-3.4.1).
+These examples are taken from [WikiPedia](https://en.wikipedia.org/wiki/Email_address#Examples), which is also based off of [RFC5322 Section 3.4.1](https://www.rfc-editor.org/rfc/rfc5322#section-3.4.1).
+
+These are valid eamples:
+
+| String                                           | Explanation                                                           |
+| ------------------------------------------------ | --------------------------------------------------------------------- |
+| `simple@example.com`                             | An EmailAddress without any special characters.                       |
+| `very.common@example.com`                        | An EmailAddress with a dot inside the local part.                     |
+| `disposable.style.email.with+symbol@example.com` | An EmailAddress with dots and a plus sign inside the local part.      |
+| `other.email-with-hyphen@example.com`            | An EmailAddress with a dot and hyphens inside the local part.         |
+| `fully-qualified-domain@example.com`             | An EmailAddress with hypehns inside the local part.                   |
+| `user.name+tag+sorting@example.com`              | An EmailAddress with a dot and plus signs inside the local part       |
+| `x@example.com`                                  | An EmailAddress with a single character in the local part.            |
+| `example-indeed@strange-example.com`             | An EmailAddress with a hyphen in both the local and domain parts.     |
+| `test/test@test.com`                             | An EmailAddress with a slash in the local part.                       |
+| `admin@mailserver1`                              | An EmailAddress with no top-level domain. This is highly discouraged. |
+| `example@s.example`                              | An EmailAddress with a dot in the domain part.                        |
+| `mailhost!username@example.org`                  | An EmailAddress with a bang in the local part.                        |
+| `user%example.com@example.org`                   | An EmailAddress with a percent sign in the local part.                |
+| `user-@example.org`                              | An EmailAddress with a hyphen in the local part.                      |
+
+These are invalid examples:
```

Comment:
I would like also to see invalid example that goes against this requirement:
> Digits `0` to `9`, provided that top-level domain names are not all-numeric (.com, .net, .org, etc.)

Reply:
Provided valid and invalid examples for top-level domain names.